11.0.Dockerfile

```diff
@@ -203,3 +212,4 @@ ONBUILD RUN mkdir -p /opt/odoo/custom/ssh \
             && sync
 ONBUILD RUN /opt/odoo/common/build && sync
 ONBUILD USER odoo
+ONBUILD VOLUME ["/var/lib/odoo"]
```

Comment:
Move above this line, after `&& sync`, just like the other dockerfiles.

Reply:
i moved this one line up now (after `ONBUILD RUN /opt/odoo/common/build && sync`)